ironfish/src/primitives/rawTransaction.ts

```diff
@@ -18,11 +17,17 @@ import { Transaction } from './transaction'
 const noteHasher = new NoteHasher()
 const MAX_MINT_OR_BURN_VALUE = BigInt(100_000_000_000_000_000n)
 
+export interface MintData {
```

Comment:
@NullSoldier asked me to move it to raw transaction. It was previously in a separate file.

Reply:
So, Yajuns change makes sense because MintDescription has an Asset, which we don't want anymore. I think this is a good place for it to go instead of primitives.

Just a note though, I said to move it somewhere in the wallet but I didn't really prescribe where, but this is a good place. https://github.com/iron-fish/ironfish/pull/3212#discussion_r1092294973

Reply:
Yeah sounds good